x-pack/test/security_solution_cypress/cypress/e2e/explore/navigation/navigation.cy.ts

```diff
@@ -92,11 +86,6 @@ describe('top-level navigation common to all pages in the Security app', { tags:
     visitWithTimeRange(TIMELINES_URL);
   });
 
-  it('navigates to the Dashboards landing page', () => {
```

Comment:
Was this failing for the ess navigation as well? just curious

Reply:
Sorry about that, I mistakenly made all of these edits only by running `yarn cypress:open:serverless`. I still have things to learn about how this file is structured, and about Cypress test runner itself.

I double checked by running `yarn cypress:open:ess`. No: there is no failure with this test enabled. Reverted: 80d934ddfee